APCAppCore/APCAppCore/Startup/APCAppDelegate.m

```diff
@@ -709,7 +711,8 @@ - (BOOL)handleUnsupportedAppVersionError:(NSError*)error networkManager:(id __un
 
 - (NSURL *)appStoreLinkURL
 {
-    return [[NSBundle mainBundle] appStoreLinkURL];
+    return [NSURL URLWithString:@""];
```

Comment:
For the future, when I do something like this I will do two things:
1. Add a #warning comment that this should not be pushed to master
2. Commit it with it's own commit and "DO NOT PUSH TO MASTER" in the commit message. Then you can use interactive rebase to remove that one commit before submitting a pull request.